### PR TITLE
Added matrix/matrices irregular

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: lib/Doctrine/Common/Inflector/Inflector.php [hunk 1]
```diff
@@ -404,7 +405,7 @@ public static function pluralize($word)
 
         if (preg_match('/(.*)\\b(' . self::$plural['cacheIrregular'] . ')$/i', $word, $regs)) {
             self::$cache['pluralize'][$word] = $regs[1] . substr($word, 0, 1) . substr(self::$plural['merged']['irregular'][strtolower($regs[2])], 1);
-            
+
```
Comment (jbrooksuk) [inline lib/Doctrine/Common/Inflector/Inflector.php]: Looks like Sublime trimmed some trailing spaces too.

**Comment by GrahamCampbell** [comment]
Missing the test cases though. ;)

**Comment by jbrooksuk** [comment]
Fancy seeing you here! :laughing:

**Comment by GrahamCampbell** [comment]
:D

File: lib/Doctrine/Common/Inflector/Inflector.php (hunk 1 above)
Comment (stof) [inline lib/Doctrine/Common/Inflector/Inflector.php]: that's OK IMO

**Comment by Ocramius** [comment]
Requires test additions

**Comment by HipsterJazzbo** [comment]
This can be closed; this case is handled by a general rule: https://github.com/doctrine/inflector/blob/master/lib/Doctrine/Common/Inflector/Inflector.php#L50

**Comment by malarzm** [comment]
Moreover, `matrix` should also fall into [this rule](https://github.com/doctrine/inflector/blob/084bc13e86ae63e653122fd53d678853a2d220e2/lib/Doctrine/Common/Inflector/Inflector.php#L49) if I'm seeing correctly. Thanks @HipsterJazzbo!

**Pull request closed without merging**